packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -122,6 +122,11 @@ export type BaseInputProps = InputLabelProps &
      * Prefix symbol to be displayed at the beginning of the input field
      */
     prefix?: string;
+    /**
+     * this is left to the components which is extending BaseInput
+     *
+     * eg: consumers can render a loader or it they could render a clear button
```

Comment:
```suggestion
     * eg: consumers can render a loader or they could render a clear button
```

Reply:
fixed in b8929d24b4e7a0d218bba7e599803658e69dc4a8